lib/community/widgets/post_card_metadata.dart

```diff
@@ -353,6 +353,78 @@ class UrlPostCardMetaData extends StatelessWidget {
   }
 }
 
+class CrossPostMetaData extends StatelessWidget {
+  final int unreadComments;
+  final int comments;
+  final bool hasBeenEdited;
+  final DateTime published;
+  final bool saved;
+  final int score;
+
+  const CrossPostMetaData({
+    super.key,
+    required this.unreadComments,
+    required this.comments,
+    required this.hasBeenEdited,
+    required this.published,
+    required this.saved,
+    required this.score,
+  });
+
+  final MaterialColor upVoteColor = Colors.orange;
```

Comment:
Add a TODO to use the customizable colours that will be introduced in this PR: #1304

Reply:
This is no longer used